### ISSUE TITLE
Feature Request: ScummVM + Touch Controls

### ISSUE THREAD

**Issue by Kwigo** [webrcade/webrcade#32]
I'd love support for ScummVM (https://www.scummvm.org/) if that's possible as well as touch controls to emulate using a mouse on mobile devices.
Thanks!

**Comment by raz0red** [comment]
Yeah, absolutely! That is one I thought about early on and intend to add support down the road.

Thanks for submitting it!

**Comment by raz0red** [comment]
Released in 0.1.7
